app/views/shared/_feed.html.erb

```diff
@@ -0,0 +1,11 @@
+
+<% @posts.each do |p| %>
```

Comment:
Posts need to always display post content and creator/author according to requirements [`REQUIRED`]

Reply:
Posts also need to be displayed in a timeline manner, according to requirements, with most recent posts displayed first. Currently, oldest post is first, take a look:

![image](https://user-images.githubusercontent.com/22228397/69247096-9b77d100-0b88-11ea-8d61-bb3e356b7d39.png)

[`REQUIRED`]